apps/io-services-cms-webapp/ServiceTopicsIngestor/function.json

```diff
@@ -0,0 +1,12 @@
+{
+  "bindings": [
+    {
+      "schedule": "0 * * * * *",
```

Comment:
is this cron expression right?

Reply:
[bd93794](https://github.com/pagopa/io-services-cms/pull/1061/commits/bd9379466a054007b05ef060a89ba3e5035181c0)